framework/resources/Datasources/join/crt_tbls_partition_by_r.ddl

```diff
@@ -0,0 +1,22 @@
+DROP TABLE IF EXISTS r_tblprtnby_intcl;
+CREATE TABLE r_tblprtnby_intcl PARTITION BY( col_int ) AS SELECT * FROM dfs.`/drill/testdata/join/typeall_r`;
+--DROP TABLE IF EXISTS r_tblprtnby_chrcl;
+--CREATE TABLE r_tblprtnby_chrcl PARTITION BY( col_chr ) AS SELECT * FROM dfs.`/drill/testdata/join/typeall_r`;
+--DROP TABLE IF EXISTS r_tblprtnby_vrchrcl1;
+--CREATE TABLE r_tblprtnby_vrchrcl1 PARTITION BY( col_vrchr1 ) AS SELECT * FROM dfs.`/drill/testdata/join/typeall_r`;
+--DROP TABLE IF EXISTS r_tblprtnby_vrchrcl2;
+--CREATE TABLE r_tblprtnby_vrchrcl2 PARTITION BY( col_vrchr2 ) AS SELECT * FROM dfs.`/drill/testdata/join/typeall_r`;
```

Comment:
They are commented for now, as there are dependent tests that fail, due to these failing CTAS statements. Will be uncommented once fix is in.